index.html

```diff
@@ -9,15 +9,20 @@
   </head>
 
   <body>
-    <header class="main-header">
+    <header>
       <nav class="nav-bar">
         <a href="index.html" class="logo">My Logo</a>
         <ul class="main-bar__items">
-          <li class="main-bar__item"><a href="">Portoflio</a></li>
+          <li class="main-bar__item"><a href="">Portfolio</a></li>
           <li class="main-bar__item"><a href="">About</a></li>
           <li class="main-bar__item"><a href="">Contact</a></li>
         </ul>
```

Comment:
- [ ] [REQUIRED] You have done a great job implementing these functionalities, however, one of the requirements states that whenever a user clicks any of the mobile menu options, the correct part of the page is displayed. Your `href` is currently empty, kindly add the correct path to each `href`. Doing this will make your project complete. Kindly implement.